extensions/stackdriver/common/constants.h

```diff
@@ -74,6 +74,8 @@ constexpr char kMonitoringEndpointKey[] = "STACKDRIVER_MONITORING_ENDPOINT";
 constexpr char kLoggingEndpointKey[] = "STACKDRIVER_LOGGING_ENDPOINT";
 constexpr char kMeshTelemetryEndpointKey[] =
     "STACKDRIVER_MESH_TELEMETRY_ENDPOINT";
+constexpr char kMonitoringExportIntervalKey[] =
+    "STACKDRIVER_MONITORING_EXPORT_INTERVAL_S";
```

Comment:
super nit: can we just expand to be `SECS` instead of `S` for enhanced readability.

Reply:
Done.